packages/contentstack-import/src/config/default.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   modules: {
     types: [
       'locales',
+      'custom-roles',
```

Comment:
Keep it just above the workflows , is there any other module other than workflow dependent on the custom roles ? is there any reason that you kept custom roles after locales